model.md

```diff
@@ -2,163 +2,162 @@
-
+| Field      | Type    | Required | Notes                                                                                          |
+| ---------- | ------- | -------- | ---------------------------------------------------------------------------------------------- |
+| struct     | keyword | true     | The name of the other [struct](#struct)                                                        |
+| labelField | string  | true     | The name of the [field](#field) whose value will identify the other [struct](#struct) instance |
+| block      | string  | true     | The name of the [block](#block) to display in the UI                                           |
 
 ### <a name="hint"></a> Hint
 
-Recommendations given to the UI about how to display the form element.  Applies to [fields](#field) and [structs](#struct)
+Recommendations given to the UI about how to display the form element. Applies to [structs](#struct), [fields](#field), and [blocks](#block).
 
-| Field       | Type     | Required | Missing Val | Notes                                       |
-|-------------|----------|----------|------------ |---------------------------------------------|
-| width       | integer  | false    |             | The recommended width of a field or block   |
+#### [Field](#field) Hints
 
+##### <a name="field-stamp"></a> Additional properties for hints of [type](#type) stamp
 
-### <a name="option"></a> Option
+| Field             | Type     | Required | Missing Val | Notes                                                        |
+| ----------------- | -------- | -------- | ----------- | ------------------------------------------------------------ |
+| headerSize        | integer  | false    |             | The size of the stamp element when display using the `h` tag |
+| displayClassNames | string[] | false    |             | The CSS class name to append to the stamp element            |
+
+#### [Block](#block) Hints
 
+##### <a name="field-linked-struct"></a> Additional properties for hints of [type](#type) linkedStruct reference
 
-| Field    | Type                     | Required | Notes                                       |
-|----------|--------------------------|----------|---------------------------------------------|
-| label    | string or [Label](#label)| true     | The display value for the option in the UI  |
-| value    | value cosistent with the field [type](#type) | true | the value associated with the option |
+| Field  | Type   | Required | Missing Val | Notes                                                                                             |
+| ------ | ------ | -------- | ----------- | ------------------------------------------------------------------------------------------------- |
+| layout | string | false    | inline      | Indicates how to layout the [Struct](#struct) instances                                           |
```

Comment:
If there is a predefined list of values for the layout, then we should have a BlockLayout type that we can link to here.  Just like we have with the `Type` type.  That is where we explain what the different layout options do, as well.

Reply:
Can we not just put those values in the notes? I'm not sure the list is ever going to change.